### ISSUE TITLE
Entities without Marking can be accessed without any permissions

### ISSUE THREAD

**Issue by SYNchroACK** [LuatixHQ/opencti#2108]
## Description

A user without a role or group (i.e. no perms) can still access objects that does not have a Marking, accessing directly the following URLs:

* http://localhost:8080/dashboard/analysis/reports/
* http://localhost:8080/dashboard/observations/observables
* http://localhost:8080/dashboard/workspaces/investigations

OpenCTI version: 5.2.4

**Comment by richard-julien** [comment]
Not sure to see any problem with that. No marking = no restriction.
That's the expected behavior. If you want to restrict access, you need to apply some markings.

**Comment by SYNchroACK** [comment]
I revisited this issue and I realized:

1. `john` with permission to view/create/update knowledge will be able to edit an observable.
2. The `admin` removes all permissions from `john` but `john` will still be logged in.
3. `john` within his session, will edit another observable successfully
4. `john` logouts and then login again and it will not be able to do anything, even if try to access directly the observable.

I did this test again and before the step 4, I deleted the [session](http://localhost:8080/dashboard/settings/accesses/sessions?) from user `john` and `john` automatically lost his access.

**Conclusion:** this behavior is okay since its related to the session. The only thing that I would suggest would be to try to reset  the user session (or something like that) to force the new permissions on users.

PS: I dont know if you guys want to close this issue and create a new one regarding session reset... it's up to you ;)
